src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -142,6 +143,49 @@ class JooqBurdenEstimateRepository(
         return parameterSetId
     }
 
+    override fun getModelRunParametersData(setId: Int): FlexibleDataTable<ModelRun>
+    {
+        val sequence = getModelRunParametersSequence(setId)
+        val headers = getModelRunParametersHeaders(setId)
+
+        return FlexibleDataTable.new(sequence, headers)
+    }
+
+    private fun getModelRunParametersHeaders(setId: Int): List<String>
+    {
+        val records = dsl.select(
+                MODEL_RUN_PARAMETER.KEY
+        )
+                .from(MODEL_RUN_PARAMETER)
+                .where(MODEL_RUN_PARAMETER.MODEL_RUN_PARAMETER_SET.eq(setId))
+                .fetch()
+                .map { mapper.mapModelParameterHeaders(it) }
+        return records
+    }
+
+    private fun getModelRunParametersSequence(setId: Int): Sequence<ModelRun>
+    {
+        val records = dsl.select(
+                MODEL_RUN_PARAMETER_VALUE.ID,
+                MODEL_RUN_PARAMETER.KEY,
+                MODEL_RUN.RUN_ID,
+                MODEL_RUN_PARAMETER_VALUE.VALUE
+        )
+                .from(MODEL_RUN_PARAMETER_VALUE)
+                .join(MODEL_RUN_PARAMETER)
+                .on(MODEL_RUN_PARAMETER_VALUE.MODEL_RUN_PARAMETER.eq(MODEL_RUN_PARAMETER.ID))
+                .join(MODEL_RUN)
+                .on(MODEL_RUN_PARAMETER_VALUE.MODEL_RUN.eq(MODEL_RUN.INTERNAL_ID))
```

Comment:
This is exactly the right logic, but can be shortened using `.joinPath` (a helper for unambiguous joins). Usage would be `.fromJoinPath(MODEL_RUN_PARAMETER_VALUE, MODEL_RUN_PARAMETER, MODEL_RUN).where...`